packages/design-system/src/css/_primitives.scss

```diff
@@ -0,0 +1,254 @@
+// Primitive Colors
+// Primitive colors shouldn't be used directly in components an other UI elements.
+// They can be only mapped to tokens.
+// Tokens should be used instead in components an other UI elements
+
+@mixin primitives {
+	// Gray
+	--prim-gray-h: 220;
+	--prim-gray-s: 30%;
+
+	--prim-gray-820: hsl(var(--prim-gray-h), 1%, 18%);
+	--prim-gray-740: hsl(var(--prim-gray-h), 2%, 26%);
+	--prim-gray-670: hsl(var(--prim-gray-h), 2%, 33%);
+	--prim-gray-540: hsl(var(--prim-gray-h), 4%, 46%);
+	--prim-gray-490: hsl(var(--prim-gray-h), 3%, 51%);
+	--prim-gray-420: hsl(var(--prim-gray-h), 4%, 58%);
+	--prim-gray-320: hsl(var(--prim-gray-h), 10%, 68%);
+	--prim-gray-200: hsl(var(--prim-gray-h), 18%, 80%);
+	--prim-gray-120: hsl(var(--prim-gray-h), 25%, 88%);
+	--prim-gray-70: hsl(var(--prim-gray-h), 32%, 93%);
+	--prim-gray-40: hsl(var(--prim-gray-h), 40%, 96%);
+	--prim-gray-10: hsl(var(--prim-gray-h), 47%, 99%);
+	--prim-gray-0: hsl(var(--prim-gray-h), 50%, 100%);
+
+	// Color Primary
+	--prim-color-primary-h: 7;
```

Comment:
🤔 I was expecting primitive to be `--prim-orange-68` for example... and then the semantic token be `--primate: var(--prim-orange-68)`